GitUI/CommandsDialogs/FormFileHistory.cs

```diff
@@ -32,17 +32,27 @@ public sealed partial class FormFileHistory : GitModuleForm
         private readonly IFullPathResolver _fullPathResolver;
         private readonly FormFileHistoryController _controller = new();
 
-        private BuildReportTabPageExtension _buildReportTabPageExtension;
+        private BuildReportTabPageExtension? _buildReportTabPageExtension;
 
         private string FileName { get; set; }
 
         [Obsolete("For VS designer and translation test only. Do not remove.")]
+#pragma warning disable CS8618 // Non-nullable field must contain a non-null value when exiting constructor. Consider declaring as nullable.
         private FormFileHistory()
+#pragma warning restore CS8618 // Non-nullable field must contain a non-null value when exiting constructor. Consider declaring as nullable.
         {
             InitializeComponent();
         }
 
-        private FormFileHistory([NotNull] GitUICommands commands)
+        /// <summary>
+        /// Open FileHistory form.
+        /// </summary>
+        /// <param name="commands">commands in the current form.</param>
+        /// <param name="fileName">name in repo of file to view.</param>
+        /// <param name="revision">initial selected commit.</param>
+        /// <param name="filterByRevision">add filter.</param>
+        /// <param name="showBlame">show blame initially instead of diff view.</param>
+        public FormFileHistory(GitUICommands commands, string fileName, GitRevision? revision = null, bool filterByRevision = false, bool showBlame = false)
```

Comment:
Merged two constructors here. Only one was used. Allows compiler verification of definite assignment to class fields.